language/move-lang/stdlib/src/lib.rs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use bytecode_verifier::{batch_verify_modules, VerifiedModule};
+use move_lang::{move_compile, shared::Address, to_bytecode::translate::CompiledUnit};
+use once_cell::sync::Lazy;
+use std::path::PathBuf;
+
+pub const STD_LIB_DIR: &str = "modules";
+pub const MOVE_EXTENSION: &str = "move";
+
+static MOVELANG_STDLIB: Lazy<Vec<VerifiedModule>> = Lazy::new(|| build_move_lang_stdlib());
+
+/// Returns a reference to the standard library built by move-lang compiler, compiled with the
+/// [default address](account_config::core_code_address).
+///
+/// The order the modules are presented in is important: later modules depend on earlier ones.
+pub fn move_lang_stdlib_modules() -> &'static [VerifiedModule] {
+    &*MOVELANG_STDLIB
+}
+
+pub fn build_move_lang_stdlib() -> Vec<VerifiedModule> {
```

Comment:
We really should duplicate/reuse/invoke this logic in the stdlib sanity check test. Would you mind doing that this PR?